cores/esp8266/debug.cpp

```diff
@@ -22,16 +22,23 @@
 #include "debug.h"
 #include "osapi.h"
 
+#ifdef DEBUG_ESP_CORE
+void iamslow (const char* what)
```

Comment:
I think you moved the fcns here to `extern "C"` with a prior PR?  Same here?

Also, just for sanity, maybe `__IAMSLOW()` or something (i.e. underscores to indicate internal use)?  Pedantic, not a big deal.

Reply:
`iamslow()` is renamed to `__iamslow()` is declared as "C" in `debug.h` (only used by the following macro)
macro `IAMSLOW()` is left untouched and used in relevant places.